portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,86 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+let catPhotoNum = 0;
+let catGalleryImage = document.getElementById("cat-gallery-image");
+let catGallerySubtitle = document.getElementById("cat-gallery-subtitle");
+let catPhotos = [
+  {"filename": "simba.jpg", "subtitle": "Simba sleeping part 1", "img": null},
+  {"filename": "simba_2.jpg", "subtitle": "Simba sleeping part 2", "img": null},
+  {"filename": "IMG_1346.JPG", "subtitle": "Simba in a box", "img": null},
+  {"filename": "IMG_1950.JPG", "subtitle": "Pepper eating food", "img": null},
+  {"filename": "IMG_3973.JPG", "subtitle": "Simba sleeping part 3", "img": null},
+  {"filename": "IMG_4355.JPG", "subtitle": "Annoyed", "img": null},
+  {"filename": "IMG_7812.JPG", "subtitle": "Looking at something", "img": null},
+  {"filename": "IMG_6657.jpeg", "subtitle": "Christmas costume", "img": null},
+];
+const MAX_WIDTH = 800;
+const MAX_HEIGHT = 600;
+
+changeCatPhoto(0);
+
+/**
+ * Switches image in cat gallery
+ * @param {number} num
+ */
+function changeCatPhoto(num) {
+  if (catPhotos[num]["img"] == null) { // Photo only needs to be loaded once
```

Comment:
For the comment style, I'm leaning to use /* */ which is indicated (but not explicitly required) in the style guide: https://google.github.io/styleguide/htmlcssguide.html#Section_Comments